src/constants.ts

```diff
@@ -0,0 +1,38 @@
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { SpecificationFile } from './hooks/validation';
+import { Context } from './hooks/context/models';
+
+export const CONTEXTFILE_PATH = path.resolve(os.homedir(), '.asyncapi');
+
+export class TestingVariables {
```

Comment:
it cannot work the way it works now. You are creating real context for the sake of the integration tests I presume, but you do it on the original `.asyncapi` file, so basically me as a developer I have bad user experience because what I have in my context is lost because of these tests.

maybe approach with `.asyncapi` should be configurable so in tests you can say the context is in some temporary folder that is in gitignore. Thoughts?

Reply:
The context service loads from the `.asyncapi` file that is in the home directory, and when I was developing, It was kind of annoying because after the tests run I always had a new .asyncapi file. If our service takes the global context file's path as an argument I can create other dummy files for testing.

Reply:
exactly, now I know where I was getting this home and code contexts from 😄 

yeah, I think we need a way to override `home/.asyncapi` for tests purposes otherwise we will get frustrated quite often